check_target_node_name : /target_node